config/provider_config.go

```diff
@@ -135,28 +141,46 @@ func (c *ProviderConfig) Init(rc *RootConfig) error {
 }
 
 func (c *ProviderConfig) Load() {
-	for key, svs := range c.Services {
-		rpcService := GetProviderService(key)
-		if rpcService == nil {
-			logger.Warnf("Service reference key %s does not exist, please check if this key "+
-				"matches your provider struct type name, or matches the returned valued of your provider struct's Reference() function."+
-				"View https://www.yuque.com/u772707/eqpff0/pqfgz3#zxdw0 for details", key)
-			continue
+	for registeredTypeName, service := range GetProviderServiceMap() {
+		serviceConfig, ok := c.Services[registeredTypeName]
+		if !ok {
+			if registeredTypeName == constant.ReflectionServiceTypeName {
+				// do not auto generate reflection server's configuration.
+				continue
+			}
+			// service doesn't config in config file, create one with default
+			logger.Warnf("Service with registeredTypeName %s does not found in configuration. Now new a default configuration to use.", registeredTypeName)
```

Comment:
Dubbogo can not find service with registeredTypeName %s in configuration. Use the default configuration instead.